back/security.py

```diff
@@ -0,0 +1,27 @@
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+
+from .models import User
+
+
+class BackAuthenticationPolicy(AuthTktAuthenticationPolicy):
```

Comment:
I believe you wouldn't need to extend the policy, just implement some callback... I will need to check